src/containers/gui.jsx

```diff
@@ -142,7 +145,8 @@ GUI.defaultProps = {
     isScratchDesktop: false,
     onStorageInit: storageInstance => storageInstance.addOfficialScratchWebStores(),
     onProjectLoaded: () => {},
-    onUpdateProjectId: () => {}
+    onUpdateProjectId: () => {},
+    onVmInit: (/* vm */) => {}
```

Comment:
Is the parameter commented out to avoid unused variable errors? Or something else?

Reply:
Oh just as a kind of light form of documentation, to show that a param does get passed to that function, but it isn't used in that default function.